cx34/cx34_registers.go

```diff
@@ -251,6 +256,7 @@ Table of registers with values that changed
 
 // Known Register values.
 const (
+	SwitchOnOff                Register = 140 // 0 = off, 1 = on
```

Comment:
Thanks for adding this. What's the source of the document in the PR description?

Reply:
Chiltrix emailed the doc to me when I asked them for info about modbus control.